package.json

```
@@ -3,8 +3,10 @@
 	"version": "1.5.0",
 	"main": "datadrop.js",
```

Shouldn't the main be changed to `./build/src/datadrop.js` ?

---

It does not really mean anything here. The `main` property defines the entry point of a module when a library is exposed publicly (that is when someone wants to import a module, they would do `require("moduleName")` where moduleName is the name of the main script in the package.json file).
As we are not delivering a module to be used outside, it does not matter to have a correct value in the main property.

For the sake of it though, I will change it to `"./build/index.js"`.
```suggestion
	"main": "./build/index.js",
```